### PR TITLE
Update for v1 EOL

### PR DESCRIPTION
**Pull request by paddycarver**
Reflect v1's EOL status in SUPPORT.md.

### CODE REVIEW

**Review by bflad — APPROVED** [review]
Looks good to me 🚀

**Review by aareet — APPROVED** [review]
Looks good, we should probably close SDKv1 issues with a link to this policy document as well

File: SUPPORT.md [hunk 1]
```diff
@@ -1,19 +1,12 @@
 # Support Policy
 
-Version 1 of the Terraform Plugin SDK is considered **deprecated**, which means
+Version 1 of the Terraform Plugin SDK is considered **end-of-life**, which means
```
Comment (aareet) [inline SUPPORT.md:3]: Just had another thought - should we include a date here @redeux ? "As of Aug 1, 2021, Version 1 of the Terraform Plugin SDK is considered end of life."

**Comment by aareet** [comment]
@paddycarver I think we can proceed with this and add the date later if needed.

**Pull request merged**